docs/share_plus/overview.mdx

```diff
@@ -20,7 +20,7 @@ on iOS.
 dependencies:
   flutter:
     sdk: flutter
-  share_plus: "^{{ plugins.share_plus }}"
```

Comment:
I am not sure this should be replaced by a hardcoded value

Reply:
Then how do you think it is better to change it?
I use this package in my codes and saw the documentation version is so old.

Reply:
I believe versions are updated when the website is redeployed, as it is generated automatically from pub.dev.